src/main/java/org/cactoos/set/SetOf.java

```diff
@@ -26,15 +26,10 @@
 import java.util.HashSet;
 import java.util.Set;
 import org.cactoos.iterable.IterableOf;
-import org.cactoos.scalar.Unchecked;
 
 /**
  * Iterable as {@link Set}.
  *
- * <p>This class should be used very carefully. You must understand that
```

Comment:
@victornoel Maybe just a few words describing this class???
Like: based on `HashSet`

Reply:
@fanifieiev thank you, it is done